### DISCUSSION TITLE
Updating tenacity?

### DISCUSSION THREAD

**Discussion by pobertraterson** [tenacityteam/tenacity-legacy#132] [Q&A] [answered]
Is there a way to update this software after it's installed?

I am unsure of a way to update it.

**Comment by fossdd** [comment, marked as answer]
Currently you have to build tenacity yourself. So updating it is done by rebuilding it. If you're using it with a package manager look how you can update tenacity with the manager.

For the progress of packaging tenacity look at: https://github.com/tenacityteam/tenacity/labels/packaging
